client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -511,21 +508,45 @@ export class CheckoutThankYouHeader extends PureComponent {
 	}
 
 	getHeaderText() {
-		return getHeading( {
-			...this.props,
-			isSearch: this.isSearch(),
-		} );
```

Comment:
This was previously refactored to a separate file as part of the redesign: https://github.com/Automattic/wp-calypso/pull/78931/files#r1254074573 Now that we're pivoting to a different approach pb5gDS-38y-p2#comment-4475, I've moved that logic back to the legacy header, so it's easier to identify what needs to be cleaned up / removed in the future.